### DISCUSSION TITLE
geopandas

### DISCUSSION THREAD

**Discussion by JCxOx** [flet-dev/flet#4421] [Packages] [answered]
Hello,

After trying Shapefile processing with fiona (not available for flet), I  tried geopandas , but it is not available as well.
It would be nice to have at least one python library able to handle geo data as shapefile.

https://pypi.org/project/geopandas/

Thanks a lot,

JC

**Comment by FeodorFitsner** [comment, marked as answer]
The package is now available for iOS and Android.

**Reply by JCxOx** [reply]
Great ! thank you
